test/local-conf-templates/conf/azkaban.properties

```diff
@@ -0,0 +1,55 @@
+# Azkaban Personalization Settings
+azkaban.name=Local
+azkaban.label=My Local Azkaban
+azkaban.color=#FF3601
+web.resource.dir=azkaban-web-server/build/install/azkaban-web-server/web
+default.timezone.id=America/Los_Angeles
+# Azkaban UserManager class
+user.manager.class=azkaban.user.XmlUserManager
+user.manager.xml.file=azkaban-solo-server/src/main/resources/conf/azkaban-users.xml
+# Loader for projects
+executor.global.properties=azkaban-solo-server/src/main/resources/conf/global.properties
+azkaban.project.dir=projects
+azkaban.execution.dir=executions
+# DB
+database.sql.scripts.dir=azkaban-db/src/main/sql
+database.check.version=true
+database.auto.update.tables=true
+#mysql
+database.type=mysql
+mysql.port=3306
+mysql.host=localhost
+mysql.database=test
+mysql.numconnections=100
+mysql.user=
```

Comment:
Users will still need to configure mysql DB. right?
Have you considered using H2 instead?

Reply:
+1 on h2. I would believe `h2` is more suitable in this case.

Reply:
I think both should be supported so that it's easy to switch. What's important for me is that I can restart the server without clearing the database, but I guess that's possible with h2 as well.

Reply:
Yes. H2 can be configured to be persistent.
And users can customize the config to use mysql.